client/assets/pages/login.scss

```diff
@@ -36,7 +36,7 @@
       font-size: map-get($font-size,  x-large) + 0.75rem;
```

Comment:
@nwochaadim Do we have a uniform measurement unit for the project?

`rem`, `em` or `px`, just curious.

Reply:
I see `rem` in some places, `em` and `px` in other places.

Reply:
@BolajiOlajide the advice from @nwochaadim  has been to use either `rem` or `em`. Perhaps he can weigh in on whether we need to standardize it to a single unit.

Reply:
This articles will help with understanding when to use `rem` and when to use `em`